nas_spec/components/schemas/issuing/card-management/cards/get-card-credentials/get-encrypted-card-credentials-response.yaml

```
@@ -0,0 +1,20 @@
+type: object
+required:
+  - encrypted_block
+properties:
+  encrypted_block:
+    type: string
+    example: "ewogICJjYXJkX2lkcyI6IFsKICAgICJjcmRfZDNvemhmNDNwY3EyeGJsZG4yZzQ1cW5iNDQiLAogICAgImNyZF9xbTI2dDUycXozeWVqam8ydDRidHF2eHR6aSIKICBdCn0="
+    description:  |
+      Encrypted Card credentials
+
+      The encrypted_block is the encrypted representation of the Card Credentials [json](http://www.json.org/) document below.
```

```suggestion
      The `encrypted_block` is the encrypted representation of the Card Credentials [json](http://www.json.org/) document.
```
Suggest removing 'below' as directional language is inaccessible.

Do we need a json.org link here? Could we link instead to the Card Credentials document?